src/modules/listClass.js

```diff
@@ -0,0 +1,153 @@
+import { forEach } from 'lodash'; // eslint-disable-line
+import Icon from '../img/icons8-menú-2-50.png';
+import trashIcon from '../img/icons8-basura-32.png';
+
+export default class taskList {
+  constructor(container) {
+    this.list = [];
+    this.container = container;
+  }
+
+  addTask(task) {
+    if (localStorage.getItem('localData') === null) {
+      localStorage.setItem('localData', JSON.stringify([]));
+      this.list.push(task);
+      localStorage.setItem('localData', JSON.stringify(this.list));
+      window.location = window.location.pathname;
+    } else {
+      this.list = JSON.parse(localStorage.getItem('localData'));

```

Comment:
the 18th line, you are getting the element from the local storage, but when you call the "render()" function, you also get the localstorage. instead, you could make it a variable outside of all the methods and add to it elements with the function, so you only update the local storage by setting it, and you modify the local storage by modifying the list already loaded instead of getting it again and again. DRY example.